pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -49,7 +49,7 @@ let
     ];
 
     buildInputs = [
-      perl
+      #perl
```

Comment:
why not drop this comment here? Imho the `--without-perl` below is quite self-explanatory.